ui/app/AppLayouts/Communities/controls/TokenPanel.qml

```diff
@@ -138,8 +138,17 @@ ColumnLayout {
             if(!addOrUpdateButton.enabled)
                 return
 
-            if(event.key === Qt.Key_Enter || event.key === Qt.Key_Return)
+            // additionally accept dot (.) and convert it to the correct decimal point char
```

Comment:
Can't use the new `StatusAmountInput` here as well? :thinking: That would reduce the code duplication